components/event-publisher-proxy/pkg/handler/handler.go

```diff
@@ -208,12 +209,12 @@ func (h *Handler) sendEventAndRecordMetrics(ctx context.Context, event *cev2even
 	result, err := h.Sender.Send(ctx, event)
 	duration := time.Since(start)
 	if err != nil {
-		h.collector.RecordError()
+		h.collector.RecordBackendError()
```

Comment:
Hi Korbinian, can we collect both `RecordBackendLatency` and `RecordBackendRequests` here and assign an http response code (5xx or 4xx) to them? as we also need the latency for the request with the error response code. and for the request number of `RecordBackendRequests`, it should also count the requests number with error response code.